test/builder/eventlistener_test.go

```diff
@@ -218,6 +219,76 @@ func TestEventListenerBuilder(t *testing.T) {
 				),
 			),
 		},
+		{
+			name: "One Trigger with Validation",
+			normal: &v1alpha1.EventListener{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "name",
+					Namespace: "namespace",
+				},
+				Spec: v1alpha1.EventListenerSpec{
+					ServiceAccountName: "serviceAccount",
+					Triggers: []v1alpha1.EventListenerTrigger{
+						v1alpha1.EventListenerTrigger{
+							TriggerValidate: &v1alpha1.TriggerValidate{
+								TaskRef: pipelinev1.TaskRef{
+									Name:       "bar",
+									Kind:       pipelinev1.NamespacedTaskKind,
+									APIVersion: "v1alpha1",
+								},
+								ServiceAccountName: "foo",
+								Params: []pipelinev1.Param{{
+									Name: "Secret",
+									Value: pipelinev1.ArrayOrString{
+										Type:      pipelinev1.ParamTypeString,
+										StringVal: "github-secret",
+									},
```

Comment:
nit: I found it a little jarring when reading the code that this Param and the next one were indented differently even though they are in the same list. It's just my personal preference, but I would prefer if this Param had the same indentation as the next one:
```
Params: []pipelinev1.Param{
  {
    Name: "Secret",
    Value: pipelinev1.ArrayOrString{
      Type:      pipelinev1.ParamTypeString,
      StringVal: "github-secret",
    },
  },
...
```